R/filters.R

```diff
@@ -119,3 +119,31 @@ filter_by_repertoire <- function(.data, .min_size) {
 filter_by_clonotype <- function(.data, .query) {
   return(list(data = .data$data, meta = .data$meta))
 }
+
+include <- function(...) {
+  args <- unname(list(...))
```

Comment:
I there a reason why you used unname? In general, I think include should have a character vector as parameter (which you in repFilter can be used in %in%, I think ), so there should be unlist

Reply:
List is the very interesting data structure, but vectors are quiet good too)

Reply:
Ellipsis object must be extracted with `list()`, and it can be named, so `unname()` is also needed; I will add `unlist()` to convert it to vector